pom.xml

```diff
@@ -189,7 +189,7 @@
         <module>xstream</module>
 
         <module>struts2</module>
-
+        <module>hibernate5</module>
   </modules>
 
-</project>
```

Comment:
There's a merge conflict with this file -- you'll need to update your branch from upstream, resolve the conflict(s),  and do another push. Not sure what the problem is exactly -- could be as simple as restoring the empty line at the end of the file.